Installer and tool chain both in Python

PressedCode opened the pull request.

LubosKuzma commented:
Hi, unfortunately this hasn't been done properly. This seems like the code has been copied into the comments of this pull request. Can you also please provide me with your full name? Thank you

vubump123 commented:
It's Minh

On Thu, 30 Nov 2023 at 12:37, LubosKuzma ***@***.***> wrote:

> Hi, unfortunately this hasn't been done properly. This seems like the code
> has been copied into the comments of this pull request. Can you also please
> provide me with your full name? Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LubosKuzma/AssemblyToolchains/pull/9#issuecomment-1834430578>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7XAL6PJWSL3OZEGCDURXMTYHDN7JAVCNFSM6AAAAAA6546AS2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMZUGQZTANJXHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

LubosKuzma commented:
Thank you

PressedCode committed:
Entire tool chain installer in python

This program is a verbatim recreation of the bash installer but rewritten in python. I am 90% sure that it works correctly but it is missing some error handling and I could not test how it would work on an arm system.

PressedCode committed:
Create pytoolchain.py

LubosKuzma commented:
Excellent, thank you
